### PR TITLE
AT_05.02_008 | Create a new Pipeline

### PR DESCRIPTION
**Pull request by Nzs1**
https://trello.com/c/AsZlf2Dg/1096-at0502008-create-a-new-pipeline

### CODE REVIEW

**Review by nadilova — APPROVED** [review]

**Review by byKosta — APPROVED** [review]

File: cypress/e2e/pipelineCreate.cy.js [hunk 1]
```diff
@@ -50,5 +54,16 @@ describe('pipelineCreate', () => {
...
+    })
+
+    it('AT_05.02_008 | Create a new Pipeline', () => {
+        cy.get('.task:first-child ').contains(homePage.sidePanelItems[0]).click()
+        cy.get('input#name').type(pipelineName.namePipeline)
+        cy.get('span.label').contains(newItemNames.projectNames[1]).click()
+        cy.get('#ok-button').click()
+        cy.get('button[name="Submit"]').click()
+        cy.get('a.model-link').contains(myView.dashboard).click()
+        cy.get('table#projectstatus').should('contain', pipelineName.namePipeline)
```
Comment (DariaMas7) [inline cypress/e2e/pipelineCreate.cy.js:67]: if we have an id as our selector we don't need to add any other tag
id is unique so the locator would be #projectstatus

**Comment by Nzs1** [comment]
[AT_05.02_008 | Create a new Pipeline](https://trello.com/c/AsZlf2Dg/1096-at0502008-create-a-new-pipeline)

File: cypress/e2e/pipelineCreate.cy.js (hunk 1 above)
Comment (nadilova) [reply cypress/e2e/pipelineCreate.cy.js:67]: unless we want to make a selector more informative?

Comment (SvetlanaVet) [reply cypress/e2e/pipelineCreate.cy.js:67]: that's very important to make an informative selector and in my opinion, it's need to add semantic sense to the selector

**Pull request merged**